### ISSUE TITLE
Is it possible to extend DynamoDb export template to export multiple tables?

### ISSUE THREAD

**Issue by payman81** [aws-samples/data-pipeline-samples#68]

**Comment by sivankumar86** [comment]
@payman81  yes, It is possible and you need to add  EmrActivity for new tables for the sequential run or hadoop activity for the parallel run. Refer: http://docs.aws.amazon.com/datapipeline/latest/DeveloperGuide/dp-object-activities.html
